src/controllers/busStop.js

```diff
@@ -0,0 +1,92 @@
+import cryptoRandomString from 'crypto-random-string';
+import model from '../database/models';
+import succesRes from '../helpers/successHandler';
+import errorRes from '../helpers/errorHandler';
+
+const { busStop } = model;
+const createBusStop = async (req, res) => {
+  try {
+    const { coordinates, sector, cell } = req.body;
+    const createdBusStopId = cryptoRandomString({ length: 5, type: 'numeric' });
+    const searchBusStop = await busStop.findAll({ where: req.body });
+    const searchBusStopId = await busStop.findAll({
+      where: { busStopId: createdBusStopId },
+    });
+    if (searchBusStop.length === 0 && searchBusStopId.length === 0) {
+      const Stop = await busStop.create({
+        coordinates,
+        sector,
+        cell,
+        busStopId: createdBusStopId,
+      });
+      return succesRes(res, 201, res.__('Bus stop created successfully'), Stop);
+    }
+    return errorRes(res, 409, res.__('Bus stop already exists'));
+  } catch (error) {
+    return errorRes(res, 500, res.__('Internal server error : ') + error.message);
+  }
+};
+
+const getBusStops = async (req, res) => {
+  try {
+    const foundBusStop = await busStop.findAll();
+    if (foundBusStop.length === 0) {
+      return errorRes(res, 404, res.__('No created busStops , Please add one'));
+    }
+    return succesRes(res, 200, res.__('All busStops'), foundBusStop);
+  } catch (error) {
+    return errorRes(res, 500, res.__('Internal server error : ') + error.message);
+  }
+};
+
+const oneStop = async (req, res) => {
+  try {
+    const busStopId = req.params;
```

Comment:
does really all the req.params equals to busStopId? check on this, we should get the busStopId from req.params not equalizing them